keyboards/handwired/evk/v1_3/config.h

```diff
@@ -39,13 +39,13 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
  *
  */
 #define MATRIX_ROW_PINS \
-    { B0, B1, B2, B3, B7, D0 }
+    { B0, B1, B2, B3, B7, D0 }  // row 0, 1, 2, ...
 #define MATRIX_COL_PINS \
-    { D1, D2, D3, C6, C7, F0, F1, F4, F5, F6, F7, B6, B5, B4, D7, D6 }
+    { D1, D2, D3, C6, C7, F0, F1, F4, F5, F6, F7, B6, B5, B4, D7, D6 }  // column 0, 1, 2, 3, ...
```

Comment:
```suggestion
    { D1, D2, D3, C6, C7, F0, F1, F4, F5, F6, F7, B6, B5, B4, D7, D6 }
```

Reply:
Hi, would it be possible to retain these comments to assist readers in the future?  
https://github.com/YangPiCui/ErgonomicVerticalKeyboard/tree/master/EVK_v1.3/ElectronicsFirmwareAndSoftware#121-handwiredevkv1_3